src/components/SubmissionsPage/Submissions.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+const useStyles = makeStyles(theme => ({
+	title: {
+		fontWeight: theme.typography.fontWeightBold,
+		paddingTop: theme.spacing(11),
+		[theme.breakpoints.down('sm')]: {
+			paddingTop: theme.spacing(4)
+		},
+		paddingBottom: theme.spacing(4)
+	},
+	info: {
+		paddingTop: theme.spacing(4)
+	},
+	link: {
+		color: theme.palette.secondary.main
+	},
+	linkClicked: {
+		color: theme.palette.secondary.dark
+	}
+}));
+
+const SubmissionsPage = () => {
+	const classes = useStyles();
+
+	const [formClicked, setFormClicked] = useState(false);
+	const [devPostClicked, setDevPostClicked] = useState(false);
+
+	const handleFormClick = () => {
+		setFormClicked(true);
+	};
+
+	const handleDevPostClick = () => {
+		setDevPostClicked(true);
+	};
+
+	return (
+		<Container maxWidth="md">
+			<Typography
+				align="left"
+				variant="h4"
+				component="h1"
+				className={classes.title}
+			>
+				Submission Info
+			</Typography>
+
+			<Typography align="left" variant="h5" component="h2">
+				Follow These 2 Final Steps to Submit Your Project:
+			</Typography>
+
+			<Typography className={classes.info}>
+				1{')'} Submit the project on the HOTH XI{' '}
+				<a
```

Comment:
Is there a specific reason we used the <a> tag here instead of MUI's <Link> tag? If not, maybe switch to <Link>

There's an example of this tag's usage in src/components/HomePage/HothDescription.js